### ISSUE TITLE
Bundled subtitles and nzbToSickbeard

### ISSUE THREAD

**Issue by rEes9P** [clinton-hall/nzbToMedia#1773]
**Describe the bug**
_I'd like to ask a question and make a suggestion. This is not a bug report._
I have been using your excellent scripts for a long time and would like to thank you profusely for your work! I suddenly realised today that, despite being a fairly competent user, I wasn't quite sure as to how to keep bundled subtitles for tv shows when using nzbToSickbeard in conjunction with SickChill. Is there an option, along the lines of the SatelliteExtensions option on VideoSort, that would leave files with specified extensions alone? I would like to keep the subtitles that are bundled with the release but don't want nzbToSickbeard to transcode the media file or make any changes to it.

Going through the many options I also felt that it would be nice to have a simple, stripped down version of nzbToSickbeard that would simply relay the download to SickChill and not have any options for transcoding etc. This would, I'd presume, meet the needs of a large number of users and take away the possibility of people mucking around and breaking things with settings such as niceness, ionice_class etc, which, after years of being a dedicated user, even I have no clue about!

This is just a suggestion and is in no way meant to be a complaint! Thanks very much again :-)

**Technical Specs**
1. Running on: Mac
2. Python version: 3.7.7
3. Download Client: NZBget
4. Intended Media Management: SickChill

**Comment by clinton-hall** [comment]
In autoProcessMEdia.cfg
```
[Extensions]
    compressedExtensions = .zip,.rar,.7z,.gz,.bz,.tar,.arj,.1,.01,.001
    mediaExtensions = .mkv,.avi,.divx,.xvid,.mov,.wmv,.mp4,.mpg,.mpeg,.vob,.iso,.m4v,.ts
    audioExtensions = .mp3, .aac, .ogg, .ape, .m4a, .asf, .wma, .flac
    metaExtensions = .nfo,.sub,.srt,.jpg,.gif
```
The sub files are defined in the metaExtensions.

To be honest, outside of transcoding, nzbToMedia doesn't have much to do with the Sub handling.

I believe what you want is actually to change the way SickChill decides what files to keep/discard...

Check SickChillS settings
https://github.com/SickChill/SickChill/wiki/Settings-explained#plugin-settings

Note the option to specify `keep associated file extensions`
If you add `,sub,srt` to this list, that should keep the original subs.

Note: SickChill also has a whole section about Subtitles, but this is geared mainly towards downloading additional Subs.

I appreciate the feedback. Re: making a simple version... what is somewhat on the cards is stripping out functions as separate modules, so you would have the basic module, then add Transcoding etc. as additional if wanted... But I don't even have a real idea as to when i will get to this...

**Comment by rEes9P** [comment]
Thanks much for your considered response, @clinton-hall. My autoProcessMedia.cfg file looks identical to the setting you mention except for `mediaExtensions = ".mkv,.avi,.divx,.xvid,.mov,.wmv,.mp4,.mpg,.mpeg,.vob,.iso,.flv,.f4v,.ts"` (with the quotes in there.) On the SickChill side of things I have `Associated file extensions = srt,sub,idx` and _Delete non associated files_ is checked. I tried to replicate the issue but couldn't and had the subtitles download fine on the couple of files I tried out so I’m not quite sure what was wrong earlier but guess everything is working as expected again!

The simpler, modular version of nzbToMedia sounds great. I look forward to using it as and when it sees the light of day.
